src/main/java/net/minecraftforge/common/ForgeMod.java

```
@@ -130,9 +143,160 @@
      * Noop biome modifier. Can be used in a biome modifier json with "type": "forge:none".
      */
     public static final RegistryObject<Codec<NoneBiomeModifier>> NONE_BIOME_MODIFIER_TYPE = BIOME_MODIFIER_SERIALIZERS.register("none", () -> Codec.unit(NoneBiomeModifier.INSTANCE));
+    private static final DeferredRegister<FluidType> VANILLA_FLUID_TYPES = DeferredRegister.create(ForgeRegistries.Keys.FLUID_TYPES, "minecraft");
+
+    public static final RegistryObject<FluidType> EMPTY_TYPE = VANILLA_FLUID_TYPES.register("empty", () ->
+            new FluidType(FluidType.Properties.create()
+                    .descriptionId("block.minecraft.air")
+                    .motionScale(1D)
+                    .canPushEntity(false)
+                    .canSwim(false)
+                    .fallDistanceModifier(1F)
+                    .pathType(null)
+                    .adjacentPathType(null)
+                    .density(0)
+                    .temperature(0)
+                    .lightLevel(0)
+                    .viscosity(0))
+            {
+                @Override
+                public void setItemMovement(ItemEntity entity)
+                {
+                    if (!entity.isNoGravity()) entity.setDeltaMovement(entity.getDeltaMovement().add(0.0D, -0.04D, 0.0D));
+                }
 
+                @Override
+                public void initializeClient(Consumer<IFluidTypeRenderProperties> consumer)
+                {
+                    consumer.accept(new IFluidTypeRenderProperties()
+                    {
+                        @Override
+                        public int getColorTint()
+                        {
+                            return 0;
+                        }
+                    });
+                }
```

Is there any reason to not just let this default to the white default of `getColorTint` and then the fluid properties defaulting to the DUMMY that otherwise matches the fluid properties for empty?